bamboo/bootstrap-integration-tests.sh

```diff
@@ -14,11 +14,15 @@ npm config set unsafe-perm true
 npm install
 . ./bamboo/set-bamboo-env-variables.sh
 
-if [[ $USE_NPM_PACKAGES == true ]]; then
-  echo "***Deploying stack with NPM packages"
-  (cd example && npm install)
+if [[ $USE_TERRAFORM_ZIPS == true ]]; then
+  echo "***Deploying stack with deployment packages"
+  cd example/cumulus-tf
+  sed -i 's/source = \"..\/..\//source = \"https:\/\/github.com\/nasa\/cumulus\/releases\/download\/$VERSION_FLAG/terraform-aws-cumulus.zip\/\//g' *.tf
```

Comment:
We do release a separate `workflow` module artifact that we are pulling into our workflows as `source = "../../tf-modules/workflow"`, though. Shouldn't the `*workflow.tf` files point at `terraform-aws-cumulus-workflow.zip` instead?

Reply:
Yes, you're right.  I punted on this after getting a 👍  on the approach until before the release. 


b4d37d1